dask/dataframe/core.py

```diff
@@ -3713,6 +3739,7 @@ def map_partitions(func, *args, **kwargs):
     """
     meta = kwargs.pop('meta', no_default)
     name = kwargs.pop('token', None)
+    transform_divisions = kwargs.pop('transform_divisions', True)
```

Comment:
Would appreciate documenting this in the docstring, especially around edge cases.

IIUC, users can expect `.map_partictions(func)` to retain divisions iff `func` is monotonic? This seems like a perfectly acceptable rule for now.

Reply:
Added a docstring entry

> IIUC, users can expect .map_partictions(func) to retain divisions iff func is monotonic? This seems like a perfectly acceptable rule for now.

 Yes, if the input is an Index object